src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/ManagedTextSection.cs

```diff
@@ -90,7 +90,7 @@ internal sealed class ManagedTextSection
         /// If set, the module contains instructions that assume a 64 bit instruction set. For example it may depend on an address being 64 bits.
         /// This may be true even if the module contains only IL instructions because of PlatformInvoke and COM interop.
         /// </summary>
-        internal bool Requires64bits => Machine == Machine.Amd64 || Machine == Machine.IA64 || Machine == Machine.Arm64;
+        internal bool Requires64bits => Machine == Machine.Amd64 || Machine == Machine.IA64 || Machine == Machine.Arm64 || Machine == Machine.LoongArch64;
```

Comment:
These checks in System.Reflection.Metadata are only needed for platforms supported by .NET Framework on Windows.

Could you please revert the changes in System.Reflection.Metadata? 

If you need these changes to make something work, let's discuss how to address it differently.

Reply:
Ok, thank you very much, I have revert the change in `ManagedTextSection.cs`. As you said, revert here will not cause any impact on linux-LA64.
But the `PEHeaderBuilder.cs` which also under the directory `System.Reflection.Metadata` is important, this will affect the packaging format for the dll under R2R.